src/Hedgehog/Hedgehog.fsproj

```diff
@@ -2,7 +2,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>netstandard1.6;netstandard2.0;net45</TargetFrameworks>
-    <Version>0.9.0</Version>
+    <Version>0.10.0</Version>
```

Comment:
@adam-becker, the most important change, of course, is this one.

Reply:
@moodmosaic At work, we've been using [GitVersion][gitversion] to grab version information from git, it works pretty nicely in my opinion.

[gitversion]: https://gitversion.net/docs/